package.json

```diff
@@ -22,6 +22,7 @@
     "develop": "webpack --mode=development --watch & webpack-dev-server --mode=development & wait",
     "build": "webpack --mode='production'",
     "prepublish": "tsc",
+    "lint": "eslint '{src,tests}/**/*.{js,ts}' --quiet --fix",
```

Comment:
Is eslint calling prettier?

Reply:
Yes prettier can be set up inside `.eslintrc` using the folloing extends command:
```
  "extends": [
    ...
    "prettier/@typescript-eslint",
    "plugin:prettier/recommended"
  ],
```